lib/opensusebasetest.pm

```diff
@@ -237,11 +237,11 @@ Inspect the YaST2 logfile checking for known issues.
 
 =cut
 sub investigate_yast2_failure {
-    my ($self) = shift;
-
+    my ($self, $args) = shift;
+    my $args->{logs_path} //= '/var/log/YaST2/';
```

Comment:
A couple of things here:
- let's remove then `$logs_path` variable which is used later and it is the same.
- update perldoc for this function

Reply:
Syntax was still in experimental phase.  Therefore [WIP] was not removed